### PR TITLE
[microNPU] re-enable network tests

### PR DESCRIPTION
**Pull request by manupa-arm**
This commit re-enables tests that
had failed due to a interrupted downloading
of the testing data.

### CODE REVIEW

**Review by lhutton1 — APPROVED** [review]
Thanks @manupa-arm, LGTM! Looks like CI hit a failing GPU test `tests/python/frontend/mxnet/test_forward.py`

**Comment by manupa-arm** [comment]
Fair enough @driazati ! did the change

File: python/tvm/relay/testing/tf.py [hunk 1]
```diff
@@ -216,32 +216,43 @@ def get_workload_official(model_url, model_sub_path):
...
-        zip_object.close()
-    else:
-        raise RuntimeError("Could not decompress the file: " + model_path)
-    return os.path.join(dir_path, model_sub_path)
+    attempts = retries + 1
+    error = None
+    for current_attempt_idx in range(attempts):
+        try:
+            model_tar_name = os.path.basename(model_url)
+            model_path = download_testdata(model_url, model_tar_name, module=["tf", "official"])
```
Comment (driazati) [inline python/tvm/relay/testing/tf.py:234]: any reason to do retries here instead of passing `retries=5` or whatever to `download_testdata`?

Comment (manupa-arm) [reply python/tvm/relay/testing/tf.py:234]: Well, now in the issue you reported the error happens in the extractall function, where it find the download to be corrupt.

So this should cover both cases

**Comment by manupa-arm** [comment]
cc : @Mousius @leandron

File: python/tvm/relay/testing/tf.py [hunk 2]
```diff
@@ -216,32 +216,43 @@ def get_workload_official(model_url, model_sub_path):
...
-
-        zip_object = zipfile.ZipFile(model_path)
-        zip_object.extractall(path=dir_path)
-        zip_object.close()
-    else:
-        raise RuntimeError("Could not decompress the file: " + model_path)
-    return os.path.join(dir_path, model_sub_path)
+    attempts = retries + 1
+    error = None
+    for current_attempt_idx in range(attempts):
```
Comment (Mousius) [inline python/tvm/relay/testing/tf.py:231]: `download_testdata` eventually calls into `download`:
https://github.com/apache/tvm/blob/1fd1b794fe3b50075ce45418546ee6999a36bc73/python/tvm/contrib/download.py#L174
which already implements a `retries` flag:
https://github.com/apache/tvm/blob/1fd1b794fe3b50075ce45418546ee6999a36bc73/python/tvm/contrib/download.py#L29

What's the reason for implementing retries again here?

**Review by Mousius — CHANGES_REQUESTED** [review]

File: python/tvm/relay/testing/tf.py (hunk 2 above)
Comment (Mousius) [reply python/tvm/relay/testing/tf.py:231]: Oh nm, saw it was due to corrupt archives above... my bad :crying_cat_face:

**Review by Mousius — APPROVED** [review]

**Comment by Mousius** [comment]
Thanks @manupa-arm, tests re-enabled :smile_cat: !

**Pull request merged**